docs/src/guides/laravel/deploy/scheduling.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Workers, Cron Jobs and Task Scheduling"
+sidebarTitle: "Scheduling Tasks"
+weight: -100
+description: How to schedule Tasks for your Laravel application.
+---
+
+Laravel offers a very convenient and flexible way of scheduling tasks. A large set of [helpers functions](https://laravel.com/docs/scheduling#schedule-frequency-options) allows the easy scheduling of Commands and Jobs.
+
+Once the scheduled tasks are defined, they need to be effectively executed at the right time and pace.
+The recommended way is a cron configuration entry running the `artisan schedule:run` command.
+
+```yaml {location=".platform.app.yaml"}
+crons:
+    # Run Laravel's scheduler every 5 minutes, which is as often as crons can run.
```

Comment:
This is true for Professional plans, but not Elite or Enterprise, as noted immediately below. So I would probably adjust the comment.